aten/src/ATen/native/LinearAlgebra.cpp

```diff
@@ -231,18 +277,61 @@ Tensor linalg_pinv(const Tensor& input, const Tensor& rcond, bool hermitian) {
     Tensor S_abs = S.abs();
     // eigenvalues are sorted in ascending order starting with negative values, we need a maximum value of abs(eigenvalues)
     Tensor max_val = S_abs.amax(/*dim=*/-1, /*keepdim=*/true);
-    Tensor S_pseudoinv = at::where(S_abs > (rcond.unsqueeze(-1) * max_val), S.reciprocal(), at::zeros({}, S.options())).to(input.dtype());
+    Tensor tol = at::max(atol.unsqueeze(-1), rtol.unsqueeze(-1) * max_val);
+    Tensor S_pseudoinv = at::where(S_abs > tol, S.reciprocal(), at::zeros({}, S.options())).to(input.dtype());
     // computes U @ diag(S_pseudoinv) @ U.conj().T
     return at::matmul(U * S_pseudoinv.unsqueeze(-2), U.conj().transpose(-2, -1));
   }
 }
 
+Tensor linalg_pinv(const Tensor& input, double atol, double rtol, bool hermitian) {
+  Tensor atol_tensor, rtol_tensor;
+  std::tie(atol_tensor, rtol_tensor) = get_atol_rtol(input, atol, rtol);
+  return at::linalg_pinv(input, atol_tensor, rtol_tensor, hermitian);
+}
+
+Tensor linalg_pinv(const Tensor& input, const Tensor& rcond, bool hermitian) {
+  // For NumPy compatibility the rcond argument is used as relative tolerance
+  TORCH_CHECK(!at::isComplexType(rcond.scalar_type()),
+              "torch.linalg.pinv: rcond tensor of complex type is not supported.");
+  auto options = input.options().dtype(ScalarType::Double);
+  return at::linalg_pinv(input, at::zeros({}, options), rcond, hermitian);
+}
+
 Tensor linalg_pinv(const Tensor& input, double rcond, bool hermitian) {
-  Tensor rcond_tensor = at::full({}, rcond, input.options().dtype(ScalarType::Double));
-  return at::linalg_pinv(input, rcond_tensor, hermitian);
+  // For NumPy compatibility the rcond argument is used as relative tolerance
```

Comment:
Same as for `matrix_rank`, is this documented?

Reply:
In this PR I added a sentence "If the value of :attr:`rcond` is specified then it's equivalent to setting `rtol=rcond`."

Reply:
We discussed with @mruberry the option of start having a section of "compatibility arguments" after the "Keyword arg:" section in the docs. This section would advertise the arguments saying "this is just here for compatibility with XXX framework". Perhaps this could be a fantastic start for this section? Add something like:
> Compatibility args:
>   rcond (float, Tensor, optional): [Numpy] alias for ``:attr:`rtol` ``.

Reply:
I added the Compatibility args section.